libcontainer/process_linux.go

```diff
@@ -120,6 +120,13 @@ func (p *setnsProcess) signal(sig os.Signal) error {
 
 func (p *setnsProcess) start() (retErr error) {
 	defer p.comm.closeParent()
+
+	if p.process.IOPriority != nil {
```

Comment:
It would be way better to add
```go
if prio == nil {
   return nil
}
```

to the beginning of `utils.SetIOPriority`, and remove the condition here and in other places that call this